sdk/lib/atl/atlwin.h

```diff
@@ -1719,6 +1730,22 @@ class CContainedWindowT : public TBase
         return TRUE;
     }
 
+    HWND UnsubclassWindow(BOOL bForce = FALSE)
+    {
+        if (!bForce)
+        {
+            // TODO:
+            return NULL;
+        }
+        CContainedWindowT<TBase> *pThis;
+        pThis = reinterpret_cast<CContainedWindowT<TBase> *>(this);
+        HWND hwndOld = pThis->m_hWnd;
+        pThis->m_hWnd = NULL;
+        ::SetWindowLongPtrW(hwndOld, GWLP_WNDPROC, m_pfnSuperWindowProc);
```

Comment:
You probably want to check if the current WNDPROC of the window is `m_thunk.GetWNDPROC()`.
If it is that (or you passed force), you can restore the `m_pfnSuperWindowProc`

Stuff like this should ideally be documented in a simple apitest, that you can write against Microsoft's ATL.